pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -164,7 +164,7 @@ var (
 	plaintextHTTPALPNs = []string{"http/1.0", "http/1.1", "h2c"}
 	mtlsHTTPALPNs      = []string{"istio-http/1.0", "istio-http/1.1", "istio-h2"}
 
-	mtlsTCPALPNs = []string{"istio"}
+	mtlsTCPALPNs = []string{"istio", "istio-peer-exchange"}
```

Comment:
This ALPN is not used.

Reply:
I see it getting used here: https://github.com/istio/istio/blob/da018731d9d63dae0692ce620fcb9b2f4d48540c/pilot/pkg/networking/core/v1alpha3/listener.go

Reply:
Oh, I see. It's for filter chain match.